File: 7kyu/(7 kyu) Complementary DNA/(7 kyu) Complementary DNA.java
public class DnaStrand {
  public static String makeComplement(String dna) {
    return dna
      .replaceAll("A", "t")
      .replaceAll("T", "a")
      .replaceAll("C", "g")
      .replaceAll("G", "c")
      .toUpperCase();
  }
}

File: 7kyu/(7 kyu) Complementary DNA/(7 kyu) Complementary DNA.py
def DNA_strand(dna):
    # #1
    # dna = dna.replace("A", "t")
    # dna = dna.replace("T", "a")
    # dna = dna.replace("C", "g")
    # dna = dna.replace("G", "c")
    
    # return dna.upper()
    # #2
    # Python 3.4
    return dna.translate(str.maketrans("ATCG","TAGC"))
